app/src/main/res/layout/nightmode.xml

```diff
@@ -23,25 +23,23 @@
 
                 <TextView
                         android:id="@+id/title"
-                        android:layout_width="wrap_content"
+                        android:layout_width="0dp"
```

Comment:
The spinner was not respecting its gravity well so I just made the title consume the rest of the white space that the spinner was not. I don't just suck.